src/components/CorporateCampaign/CampaignThanks.vue

```diff
@@ -160,6 +161,14 @@ export default {
 				userHasLentBefore(hasLentBefore);
 				userHasDepositBefore(hasDepositBefore);
 
+				// MARS-246 Hotjar user attributes
+				setHotJarUserAttributes({
+					userId: this.userId,
```

Comment:
`this.userId` isn't defined on this component, you should however be able to check for it on the `lender` data property that is updated...something like `this.lender?.my?.userAccount?.id` ... need to check the actual page data structure at this point in time.

You can checkout on `/cc/kiva-universal` without a promo code to test this behavior.